special-pages/pages/new-tab/app/remote-messaging-framework/RemoteMessagingFramework.module.css

```diff
@@ -0,0 +1,195 @@
+.root {
+    --ntp-rmf-surface-background-color: rgba(0, 0, 0, .06);
+    background: var(--ntp-rmf-surface-background-color);
+    padding: 14px var(--sp-8) 14px var(--sp-4);
+    border-radius: var(--sp-3);
+    position: relative;
+    display: flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    font-family: system-ui, BlinkMacSystemFont, 'Segoe UI', Roboto;
+    color: var(--color-black);
+
+
+    &.icon {
+        padding-left: var(--sp-2);
+    }
+
+    @media screen and (prefers-color-scheme: dark) {
+        background-color: var(--color-white-at-6);
+        color: var(--color-white-at-84);
+    }
+}
+
+.iconBlock {
+    margin-right: var(--sp-2);
+    width: var(--sp-12);
+    min-width: var(--sp-12);
+}
+
+.content {
+    flex-grow: 1;
+    font-size: calc(14 * var(--px-in-rem));
+}
+
+.title {
+    font-weight: var(--title-2-font-weight);
+    line-height: normal;
+    margin-bottom: var(--sp-2);
+}
+
+.description {
+    line-height: 1.25rem;
+    margin-bottom: var(--sp-3);
```

Comment:
I think the margin-bottom curse has found us! the small+medium layouts look affected https://deploy-preview-1114--content-scope-scripts.netlify.app/build/pages/new-tab/?rmf=small

Reply:
fix pushed